doc/specs/SCOOBE.md

```diff
@@ -0,0 +1,185 @@
+
+**3.1.2. Page Content**
+
+| **No.** | **Requirement** | **Pri** |
+| --- | --- | --- |
+|1. | The SCOOBE dialog should display the version of PowerToys the user has installed, as shown in figures 5.1.1. | P0 |
+|2. | The SCOOBE dialog information should be grouped into two sections: &quot;New Features &amp; Improvements&quot; and &quot;Bugfixes Highlights&quot;. | P0 |
+|3. | The &quot;New Features &amp; Improvements&quot; section should contain information related to end user functionality that has been added or updated. | P0 |
+|4. | The &quot;New Features &amp; Improvements&quot; section should be subdivided by the relevant utilities updated (i.e. Color Picker, FancyZones, etc.). See figure 5.1.1 and section 5.1.3 for examples. | P1 |
+|5. | The &quot;Bugfixes Highlights&quot; section should contain information related to noteworthy issues/errors that were corrected. | P0 |
+|6. | If there are relevant visuals, they should be included with the information text. See figure 5.1.1. | P1 |
+|7. | For multi-version updates (i.e. v0.29 to v0.35), combine the information from each version&#39;s updates and display them as a single &quot;New Features &amp; Improvements&quot; section and &quot;Bugfixes Highlights&quot; section. See section 5.1.3 for an example. | P0 |
+|8. | The SCOOBE dialog should be scrollable if needed to fit all the content. | P0 |
+|10. | The bottom of the SCOOBE dialog should include a link to the PowerToys releases page for a complete list of versions and their release notes ([Releases · microsoft/PowerToys (github.com)](https://github.com/microsoft/PowerToys/releases)). | P1 |
+
+**3.2. Open Discussion**
+
+- How do we localize content like keystrokes and visuals?
+
+- How do we make keystrokes/activation phrases stand out?
+
+- How do we handle displaying information for scenarios where a user is updating over multiple versions (ex. v0.29 -> v0.35) for which a change was made but later regressed/altered. For instance, a new FancyZones UX update in v0.31, that was again changed in v0.35?
+
+- Should we store SCOOBE information for all versions of PowerToys? If not, how far back should we maintain information?
+
+## 4. Measure Requirements
+
+| **No.** | **Requirement** | **Implication** | **Pri** |
+| --- | --- | --- | --- |
+|1. | Date/Time of first-run following upgrade | Helps to categorize usage and retention trends across users who've been exposed to SCOOBE. | P0 |
+|2. | SCOOBE section viewed | Used to measure activation of the SCOOBE dialog. Should be 100% for the current version of PowerToys following SCOOBE&#39;s inclusion. | P0 |
+|3. | Accesses to linked documentation | Used to gauge interest in user&#39;s desire to learn more about the PowerToys described. | P1 |
+|4. | Access to linked settings pages | Used to gauge whether the settings presented to users in the dialog are sufficient for user needs. | P1 |
```

Comment:
It is unclear what this is since the content is dynamic, how would SCOOBE detect that the user has clicked on link to the settings page for a particular module?
This requires require a full specification.